site/gulpfile.js

```diff
@@ -1,15 +1,18 @@
 var gulp = require('gulp');
 var elixir = require('laravel-elixir');
+var gutils = require('gulp-util');
```

Comment:
`gulp-util` helps us rend in command line parameters and use them in our gulp file.